src/client/runner/api/actions.js

```diff
@@ -751,16 +731,13 @@ export function hover (what, options) {
 
                 options = options || {};
 
-                var hoverOptions = new MouseOptions();
                 var { offsetX, offsetY } = getOffsetOptions(element, options.offsetX, options.offsetY);
 
-                hoverOptions.offsetX = offsetX;
-                hoverOptions.offsetY = offsetY;
-
-                hoverOptions.modifiers.ctrl  = options.ctrl;
-                hoverOptions.modifiers.alt   = options.alt;
-                hoverOptions.modifiers.shift = options.shift;
-                hoverOptions.modifiers.meta  = options.meta;
+                var hoverOptions = new MouseOptions({
+                    offsetX,
```

Comment:
Maybe do we need to rename it from
` var hoverOptions = new MouseOptions({`
to
` var mouseOptions = new MouseOptions({`